### ISSUE TITLE
After upgrade to Windows 11, `wsl --install` command is not enough. Asks for additional parameters.

### ISSUE THREAD

**Issue by arnisjuraga** [MicrosoftDocs/WSL#1410]
In documentation is said, that latest Windows versions can install WSL2 by running simple `wsl --install` 

When I did this on Windows 11 (After upgrade from Windows 10 with installed wsl already), command is not working. 
Output:

``` 
C:\Users\Arnis>wsl --install
Copyright (c) Microsoft Corporation. All rights reserved.

Usage: wsl.exe [Argument] [Options...] [CommandLine]

Arguments for running Linux binaries:

    If no command line is provided, wsl.exe launches the default shell.

    --exec, -e <CommandLine>
        Execute the specified command without using the default Linux shell.

    --
        Pass the remaining command line as is.

...
``` 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0cc9a77f-e6a1-a719-c337-7ff7506a1ac8
* Version Independent ID: 7ed8fb81-5b8f-4d56-7b89-bf600ce6080a
* Content: [Install WSL](https://docs.microsoft.com/en-us/windows/wsl/install)
* Content Source: [WSL/install.md](https://github.com/MicrosoftDocs/wsl/blob/main/WSL/install.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by crramirez** [comment]
Hello,

This command works for installing wsl. As you said in the description you have wsl already installed. This is why is not working. Uninstall wsl and install it again with wsl --install

Regards

**Comment by arnisjuraga** [comment]
To be honest, I don't want to "uninstall" it. 
I have upgraded Windows and I want to be able to continue to use the same WSL2 distro I have been using before.
Is there safe way to do that, by keeping my old distro filesystem and configuration at least?

**Comment by craigloewen-msft** [comment]
You can use `wsl --install -d <DistroName>` to install further distros, since the base layers of WSL are already installed. We've added better output for this in the latest WSL version. 

Please see the commands here:
https://docs.microsoft.com/en-us/windows/wsl/install#change-the-default-linux-distribution-installed

And you can run `wsl -l -v` to get a list of what distros you already have installed.

Please try running `wsl.exe` to go into a WSL instance. 

Thanks for filing!

**Comment by crramirez** [comment]
@arnisjuraga I don't quite understand your question. If you already had a working distro in Windows 10, after the upgrade to Windows 11 you don't need to do anything to continue using it. Maybe you want to upgrade WSL? In that case use `wsl --update`

**Comment by arnisjuraga** [comment]
1. Yes, I had WSL2 installed in Windows 10
```
C:\WINDOWS\system32>wsl --list
Windows Subsystem for Linux Distributions:
Ubuntu-18.04 (Default)
```

2. Now, I am sorry, it looks like I forgot, that I needed to upgrade BIOS on my MB for Windows 11 to be compatible.
Probably that changed my BIOS settings and after Windows 11 upgrade WSL2 stopped working.
I will check my BIOS settings later. 
I have already uninstalled "Windows Subsystem for Linux" from "Windows Features" and then installed it again by `wsl --install` but it didn't help, of course, if BIOS settings are wrong.

3. But still - in this scenario, it would be good to have `wsl --install` output more descriptive in this case.

I will check my BIOS settings
